x-pack/plugins/lens/server/migrations.test.ts

```diff
@@ -424,9 +424,14 @@ describe('Lens migrations', () => {
       type: 'lens',
     };
 
-    it('should remove expression', () => {
+    it('should remove search context from the expression', () => {
       const result = migrations['7.10.0'](example, context);
-      expect(result.attributes.expression).toBeUndefined();
+      expect(result.attributes.expression).not.toContain('kibana_context');
+    });
+
+    it('should include a variable instead of index pattern id', () => {
+      const result = migrations['7.10.0'](example, context);
+      expect(result.attributes.expression).toContain('index={var name=');
```

Comment:
These tests are intentionally failing for now, because they aren't complete. The expression needs to be migrated.